Cargo.toml

```diff
@@ -14,7 +14,7 @@ tendermint = { version = "0.40.0", default-features = false }
 tendermint-proto = "0.40.0"
 
 libp2p = "0.54.0"
-nmt-rs = "0.2.1"
+nmt-rs = {version = "0.2.1", features=["serde"]}
```

Comment:
a nitpick, could you instead add the feature flag only in `celestia-types`?

Reply:
+ please add spaces around inside curly braces. it's not on ci unfortunately (yet) but we format with taplo

Reply:
> a nitpick, could you instead add the feature flag only in `celestia-types`?

is this even possible? how do i do that

Reply:
Something like `nmt-rs = { workspace = true, features = ["serde"] }` in celestia types cargo toml should do. It's likely gonna have same effect as your current change but if in future we add a crate that doesn't depend on celestia types, it won't automatically inherit this feature

Reply:
Tendermint dep is good example of this pattern here

Reply:
ok got it